src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/WorkerBalanceOperations.java

```diff
@@ -70,15 +70,13 @@ public boolean addCredit(int workerID, int amount, int experiment) {
      * adds a debit to the balance-sheet of the workers wages.
      * @param workerID the primary key of the worker the debit belongs to
      * @param amount the amount that got payed
```

Comment:
not my package, but i think if i give as param a positive amount, this amount gets subtracted (thats the way i used this method)